### PR TITLE
Add boilerplate

### PR DESCRIPTION
**Pull request by amanregu**

### CODE REVIEW

File: feature/githubRepoSearch/.babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties",
+      "@babel/plugin-transform-react-jsx",
+      "@babel/plugin-transform-runtime"
+    ]
+  }
+
```
Comment (srijan-kapoor) [inline feature/githubRepoSearch/.babelrc]: Please add an empty line at the end of each file.

File: feature/githubRepoSearch/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const router = express.Router();
+
+/* GET home page. */
```
Comment (srijan-kapoor) [inline feature/githubRepoSearch/routes/index.js]: This comment can be avoided.

File: feature/githubRepoSearch/routes/users.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const router = express.Router();
+
+/* GET users listing. */
```
Comment (srijan-kapoor) [inline feature/githubRepoSearch/routes/users.js]: Same here.

**Review by srijan-kapoor — CHANGES_REQUESTED** [review]
Looks good.

Made some minor comments. Please look into them.

File: feature/githubRepoSearch/.babelrc (hunk 1 above)
Comment (amanregu) [reply feature/githubRepoSearch/.babelrc]: Done. I will keep this in mind for the future.

File: feature/githubRepoSearch/routes/index.js (hunk 2 above)
Comment (amanregu) [reply feature/githubRepoSearch/routes/index.js]: I used express-generator and this was the default boilerplate. I have removed the comments now.

File: feature/githubRepoSearch/routes/users.js (hunk 3 above)
Comment (amanregu) [reply feature/githubRepoSearch/routes/users.js]: Done.

**Pull request merged**